### PR TITLE
Add Gergoplex keymap for tgrosinger

### PR DESCRIPTION
**Pull request by tgrosinger**
## Description

Adding a new keymap to the gboards/gergoplex keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

### CODE REVIEW

**Comment by tgrosinger** [comment]
Please go ahead an merge if approved. Thank you.

File: keyboards/gboards/gergoplex/keymaps/tgrosinger/keymap.c [hunk 1]
```diff
@@ -0,0 +1,270 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/gboards/gergoplex/keymaps/tgrosinger/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/gboards/gergoplex/keymaps/tgrosinger/keymap.c (hunk 1 above)
Comment (tgrosinger) [reply keyboards/gboards/gergoplex/keymaps/tgrosinger/keymap.c:17]: Added license in new commit.

**Review by drashna — APPROVED** [review]

**Review by noroadsleft — APPROVED** [review]
Looks good to me.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**